src/context.spec.js

```diff
@@ -192,4 +192,12 @@ describe('ResourceContext', function () {
     $httpBackend.flush();
     expect(ResourceContext.busyRequests).toBe(0);
   });
+
+  it('gets synced resources on refresh', function () {
+    resource.$syncTime = 1;
+    $httpBackend.expectGET(resource.$uri, {'Accept': 'application/json'})
+      .respond('{"name": "John"}', {'Content-Type': 'application/json'});
+    context.refresh();
+    $httpBackend.flush();
+  });
```

Comment:
Is het interessant om te weten dat de requests slagen (i.e. de data in de resource terechtgekomen is)?